styles/getMargin.ts

```diff
@@ -0,0 +1,21 @@
+interface GetMarginProps {
+  marginTop: number | 'auto';
+  marginRight: number | 'auto';
+  marginBottom: number | 'auto';
+  marginLeft: number | 'auto';
+}
+
+export const getMargin = ({
+  marginTop,
```

Comment:
아래 파라미터들에게 기본 값을 준다면 StartButton에 `marginLeft={'auto'}`를 주지 않아도 될 것 같아요!
```suggestion
  marginTop = 'auto',
```

Reply:
이미 버튼 컴포넌트의 property로 기본값을 주고있습니다!